beacon-chain/rpc/service.go

```diff
@@ -172,41 +170,25 @@ func (s *Service) Start() {
 		log.Warn("You are using an insecure gRPC connection! Provide a certificate and key to connect securely")
 	}
 	s.grpcServer = grpc.NewServer(opts...)
-
-	proposerServer := &proposer.Server{
+	validatorServer := &validator.Server{
+		Ctx:                    s.ctx,
 		BeaconDB:               s.beaconDB,
+		AttestationCache:       cache.NewAttestationCache(),
+		AttPool:                s.attestationsPool,
 		HeadFetcher:            s.headFetcher,
-		BlockReceiver:          s.blockReceiver,
+		ForkFetcher:            s.forkFetcher,
+		CanonicalStateChan:     s.canonicalStateChan,
+		BlockFetcher:           s.powChainService,
+		DepositFetcher:         s.depositFetcher,
 		ChainStartFetcher:      s.chainStartFetcher,
 		Eth1InfoFetcher:        s.powChainService,
-		Eth1BlockFetcher:       s.powChainService,
+		SyncChecker:            s.syncService,
+		StateNotifier:          s.stateNotifier,
+		P2P:                    s.p2p,
+		BlockReceiver:          s.blockReceiver,
 		MockEth1Votes:          s.mockEth1Votes,
-		AttPool:                s.attestationsPool,
-		CanonicalStateChan:     s.canonicalStateChan,
-		DepositFetcher:         s.depositFetcher,
+		Eth1BlockFetcher:       s.powChainService,
 		PendingDepositsFetcher: s.pendingDepositFetcher,
-		SyncChecker:            s.syncService,
-	}
-	attesterServer := &attester.Server{
```

Comment:
No need anymore for a separate attester/proposer server! we can have everything in the validator server itself